my-app/src/components/footer/footer.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './footer.css';
+
+const Footer = () => {
+
+    return (
+        <div className="footer">
+            <div className="container">
+                <div className="block-left">
```

Comment:
data inside "block-left" move to constants.js and use .map() methods (in this case it will be object constant -> Object.entries, inside - .map())

Reply:
`const data = {
  Categories: [
    { href: "", title: "" }
  ]
};`

`Object.entries(([key, entry]) => <div>
  <div>{key}</div>  
  <div>{entry.map((item) => ...)}</div>  
</div>)`